cmd/cluster-agent/api/v1/install.go

```diff
@@ -33,10 +37,64 @@ func init() {
 
 // Install registers v1 API endpoints
 func Install(r *mux.Router, sc clusteragent.ServerContext) {
+	// The /metadata endpoints are deprecated. They will be removed as of 1.0.
+	// Agents < 6.5.0 are using /metadata.
 	r.HandleFunc("/metadata/{nodeName}/{ns}/{podName}", getPodMetadata).Methods("GET")
 	r.HandleFunc("/metadata/{nodeName}", getNodeMetadata).Methods("GET")
 	r.HandleFunc("/metadata", getAllMetadata).Methods("GET")
+	r.HandleFunc("/tags/pod/{nodeName}/{ns}/{podName}", getPodMetadata).Methods("GET")
+	r.HandleFunc("/tags/pod/{nodeName}", getNodeMetadata).Methods("GET")
+	r.HandleFunc("/tags", getAllMetadata).Methods("GET")
+	r.HandleFunc("/tags/node/{nodeName}", getNodeMeta).Methods("GET")
 	installClusterCheckEndpoints(r, sc)
+
+}
+
+// getNodeMeta is only used when the node agent hits the DCA for the list o
+func getNodeMeta(w http.ResponseWriter, r *http.Request) {
+	/*
+		Input
+			localhost:5001/api/v1/tags/nodes/localhost
+		Outputs
+			Status: 200
+			Returns: []string
+			Example: ["label1:value1", "label2:value2"]
+
+			Status: 404
+			Returns: string
+			Example: 404 page not found
+
+			Status: 500
+			Returns: string
+			Example: "no cached metadata found for the node localhost"
+	*/
+	metadataRequestsCounter.Incr(1)
+	metadataRequestsPerSecond.Set(metadataRequestsCounter.Rate())
```

Comment:
You may need to rebase since I removed the counters for stats like `metadataRequestsCounter`.